pages/common/adb-shell.md

```diff
@@ -0,0 +1,32 @@
+# adb shell
+
+> Android Debug Bridge Shell: Run remote shell command with an Android emulator instance or connected Android devices.
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
> Android Debug Bridge Shell: Run remote shell commands on an Android emulator instance or connected Android devices.
```